pydap_extras/handlers/pcic.py

```diff
@@ -290,20 +329,15 @@ def get_full_query(self, stn_id, sesh):
         return sesh.execute(query_string).first()[0]
 
     def get_vars(self, stn_id, sesh):
-        """Makes a database query to retrieve all of the climatological variables for a particular station"""
+        """Retrieves all climatological variables for a particular station"""
         q = (
             sesh.query(Variable)
-            .join(Network)
-            .join(Station)
-            .join(History)
-            .join(VarsPerHistory)
+            .join(VarsPerHistory, VarsPerHistory.vars_id == Variable.id)
```

Comment:
This is maybe beyond the scope of this PR, but I do notice that `get_vars()` is nearly identical in each of the two subclasses save for one string ("climatology" vs. "observation"). Do we want to include a small refactoring detour here?

Reply:
Yep, that was the point of a TODO I put in. Will do.